client/verta/tests/test_cli/test_endpoint.py

```diff
@@ -78,10 +78,10 @@ def test_canary_update_endpoint(self, client, created_endpoints, experiment_run,
         experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
         experiment_run.log_requirements(['scikit-learn'])
 
-        canary_rule = '{"rule": "latency", "rule_parameters": \
-        [{"name": "latency_avg", "value": "0.8"}]}'
-        canary_rule_2 = '{"rule": "error_rate", "rule_parameters": \
-        [{"name": "error_rate", "value": "0.8"}]}'
+        canary_rule = '{"rule": "latency_avg_max", "rule_parameters": \
+        [{"name": "threshold", "value": "0.8"}]}'
+        canary_rule_2 = '{"rule": "error_4xx_rate", "rule_parameters": \
+        [{"name": "threshold", "value": "0.8"}]}'
```

Comment:
While I'm nitpicking: the line break operator `\` is strongly discouraged in Python (especially to split a string omg)

Here, using
```python
json.dumps({
    ...
})
```
might be a good substitute, even if it takes up more lines

Reply:
Fixed.